packages/sui-bundler/package.json

```diff
@@ -51,7 +51,7 @@
     "strip-ansi": "6.0.1",
     "style-loader": "3.3.1",
     "url": "0.11.0",
-    "webpack-dev-server": "4.10.0",
+    "webpack-dev-server": "5.0.4",
```

Comment:
@andresz1 You maybe know better if this change has important implications in our implementation, here are the breaking changes: https://github.com/webpack/webpack-dev-server/blob/master/migration-v5.md

Reply:
I have just used the version 4.15.2 which is just a patched version fixing the vulnerability. This should work without any problems.